examples/tutorials/03_vision/01_light_when_face.py

```diff
@@ -21,13 +21,12 @@
 '''
 
 import asyncio
-import sys
 import time
 
 import cozmo
 
 
-def light_when_face(robot):
+def light_when_face(robot: cozmo.robot.Robot):
```

Comment:
run_program automatically wraps the KeyboardInterrupt try (although with tkviewer programs it doesn't help too much as tk usually gets the exception)